src/Symfony/Bridge/PhpUnit/CoverageListener.php

```diff
@@ -29,7 +29,7 @@ class CoverageListener implements TestListener
     public function __construct(callable $sutFqcnResolver = null, bool $warningOnSutNotFound = false)
     {
         $this->sutFqcnResolver = $sutFqcnResolver ?? static function (Test $test): ?string {
-            $class = \get_class($test);
+            $class = $test::class;
```

Comment:
Does this work on PHP 7.1 ? The PHPUnit Bridge is special in term on min supported PHP version.

Reply:
thanks for noticing, let's move discussion to https://github.com/symfony/symfony/pull/52117/commits/2b03e8322fa247da90ee6af144af9993dd75e773#r1363446539